cmd/rm-main.go

```diff
@@ -245,7 +246,7 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, older int) erro
 		// prefixes as well which unifies the S3 listing with folder listing - however
 		// these prefixes are not actual objects on s3 and the content.Time defaults to
 		// zero. Filter these out to prevent erroneous rmMessage being printed to console.
-		if !content.Time.IsZero() {
+		if !(content.Time.IsZero() && strings.HasSuffix(urlString, string(filepath.Separator))) {
```

Comment:
S3 service sends `/` as separator whereas on windows `\`. We would need to check whether this works on windows or not.